.github/workflows/python-client.yml

```diff
@@ -28,22 +28,25 @@ on:
       - 'scripts/build-python-client.sh'
       - 'makefile'
       - '.github/workflows/python-tests/*'
+      - './magefiles/tests.go'
+      - './magefiles/python.go'
 
 jobs:
   python-client-tox:
     runs-on: ubuntu-22.04
     strategy:
+      fail-fast: false
```

Comment:
Having all matrix combinations run to completion is always useful to avoid incrementally uncovering issues in the CI.